charts/nd-common/templates/_datadog.tpl

```diff
@@ -28,6 +28,24 @@ ad.datadoghq.com/{{ .Chart.Name }}.instances: |-
   ]
 {{- end }}
 
+{{/*
+This is datalog logging configuration. We take the .Values.scrapeLogs and
+.Values.scrapeLogsProcessingRules map and convert into list of objects converted
+into json supported by datadog config. If source and service tag values not provided
+we add default values to it.
+*/}}
+
+{{- if and .Values.datadog.enabled .Values.datadog.scrapeLogs.enabled }}
+ad.datadoghq.com/{{ .Release.Name }}.logs: |-
```

Comment:
`.Release.Name` is incorrect... it might sometimes work, but the right thing to use is the `.Chart.Name` that we use to actually name the container:
![image](https://user-images.githubusercontent.com/768067/159733551-8bcbca70-3d75-424e-961b-bf49d27410c6.png)

Reply:
done